tests/test_spans.py

```diff
@@ -124,11 +188,30 @@ def test_step_endpoint() -> str:
     provider.add_span_processor(span_processor)
     dbos_tracer.set_provider(provider)
 
+    # Set up in-memory log exporter
+    log_exporter = InMemoryLogExporter()  # type: ignore
+    log_processor = BatchLogRecordProcessor(log_exporter)
+    log_provider = LoggerProvider()
+    log_provider.add_log_record_processor(log_processor)
+    set_logger_provider(log_provider)
+    dbos_logger.addHandler(LoggingHandler(logger_provider=log_provider))
+
     client = TestClient(app)
     response = client.get("/step")
     assert response.status_code == 200
     assert response.text == '"test"'
 
+    log_processor.force_flush(timeout_millis=5000)
+    logs = log_exporter.get_finished_logs()
+    assert len(logs) == 1
+    assert logs[0].log_record.attributes is not None
+    assert (
+        logs[0].log_record.attributes["applicationVersion"] == GlobalParams.app_version
+    )
+    assert logs[0].log_record.span_id is not None and logs[0].log_record.span_id > 0
```

Comment:
Shouldn't we check that the log span ID and trace ID match those of the appropriate trace and span below?